edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/AgentTests.cs

```diff
@@ -109,7 +109,7 @@ public async void ReconcileAsyncOnEmptyPlan()
                 new SystemModules(null, null),
                 new Dictionary<string, IModule>
                 {
-                    { "mod1", new TestModule("mod1", "1.0", "docker", ModuleStatus.Running, new TestConfig("boo"), RestartPolicy.OnUnhealthy, new ConfigurationInfo("1"), null) }
+                    { "mod1", new TestModule("mod1", "1.0", "docker", ModuleStatus.Running, new TestConfig("boo"), RestartPolicy.OnUnhealthy, PullPolicy.Always, new ConfigurationInfo("1"), null) }
```

Comment:
I think we should leave current tests unchanged to test deployment conditions (where it is not specified), and add separate tests for the case where it is specified.

Reply:
Edited tests to use a combination of the different pull policy enum types. The None value will capture the cases where a value of pull policy wasn't specified at all.